modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-production-deployment.adoc

```diff
@@ -73,16 +73,16 @@ spec:
 
 == Managed vs bare-metal
 
-You can run Redpanda on managed Kubernetes platforms as well as in bare-metal environments. Managed Kubernetes services offer ease of deployment and maintenance, bare-metal environments provide complete control and potential cost efficiencies.
+You can run Redpanda on managed Kubernetes services as well as in bare-metal environments. Managed Kubernetes services offer simpler deployment and maintenance, bare-metal environments provide complete control and potential cost efficiencies.
```

Comment:
You can run Redpanda on managed Kubernetes services as well as in bare-metal environments. Managed Kubernetes services offer simpler deployment and maintenance, while bare-metal environments provide complete control and potential cost efficiencies.